catalog/core/catalog-core-impl/filter-proxy/src/main/java/ddf/catalog/filter/proxy/builder/GeotoolsBuilder.java

```diff
@@ -44,9 +46,7 @@
 import ddf.catalog.impl.filter.JTSGeometryWrapper;
 
 /**
- *
  * @author michael.menousek@lmco.com
```

Comment:
❓ Is there value in leaving this `author` tag, especially as it is a defunct email address?